runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsOverlay.java

```diff
@@ -113,7 +119,20 @@ public Dimension render(Graphics2D graphics)
 			color = Color.RED;
 			ClanMemberManager clanMemberManager = client.getClanMemberManager();
 			FontMetrics metrics = graphics.getFontMetrics();
-			String worldString = "W" + client.getWorld();
+
+			String worldString;
+			WorldResult worldResult = worldService.getWorlds();
+			if (worldResult != null)
+			{
+				World currentWorld = worldResult.findWorld(client.getWorld());
+				String worldRegion = getRegion(currentWorld.getLocation());
+				worldString = "W" + currentWorld.getId() + " (" + worldRegion + ")";
+			}
+			else
+			{
+				worldString = "W" + client.getWorld();
+			}
```

Comment:
Rather than effectively writing the code for the start of the world string twice, append the region string on a successful world service lookup.
```suggestion
			String worldString = "W" + client.getWorld();

			WorldResult worldResult = worldService.getWorlds();
			if (worldResult != null)
			{
				String worldRegion = getRegion(currentWorld.getLocation());
				worldString += " (" + worldRegion + ')';
			}
```